includes/database.inc

```diff
@@ -414,6 +421,12 @@ function db_prefix_tables($sql) {
   return $sql;
 }
 
+/**
+ * Active database connection name.
+ */
+global $db_active_name;
```

Comment:
Is this `global` declaration necessary? I suspect that just declaring the variable at the top-level will make it global.

Reply:
No, I don't think it is necessary in this case.

But it a common practice to declare global variable before its initialization:

1) It makes things more obvious.
2) This will keep variable global even if this file will be included in function context (like in db_set_active() for example).

Reply:
Ok! Let's just leave it. It matches the other global we added earlier too.